### PR TITLE
DB-10493 fix NPE when insert with 1 column into 2 column

### PR DESCRIPTION
**Pull request by martinrupp**

### CODE REVIEW

**Comment by martinrupp** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Review by dgomezferro — APPROVED** [review]

File: splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/InsertOperationIT.java [hunk 1]
```diff
@@ -466,13 +470,15 @@ public void testInsertFromSubOperation() throws Exception {
...
+    @SuppressFBWarnings("RV_RETURN_VALUE_IGNORED_BAD_PRACTICE") // delete/createNewFile
     @Test
     public void testInsertBlob() throws Exception {
-        InputStream fin = new FileInputStream(getResourceDirectory() + "order_line_500K.csv");
-        PreparedStatement ps = methodWatcher.prepareStatement("insert into FILES (name, doc) values (?,?)");
-        ps.setString(1, "csv_file");
-        ps.setBinaryStream(2, fin);
-        ps.execute();
+        try(InputStream fin = new FileInputStream(getResourceDirectory() + "order_line_500K.csv")) {
+            PreparedStatement ps = methodWatcher.prepareStatement("insert into FILES (name, doc) values (?,?)");
```
Comment (arnaud-splice) [inline splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/InsertOperationIT.java:477]: this ps and below rs should probably be closed, or surrounded by a try-with-resources, but it's unrelated to your commit

**Review by arnaud-splice — APPROVED** [review]

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

**Review by yxia92 — APPROVED** [review]
+1

File: splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/InsertOperationIT.java (hunk 1 above)
Comment (martinrupp) [reply splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/InsertOperationIT.java:477]: methodWatcher.prepareStatement is keeping a list of statements and will close them automatically. Same btw for methodWatcher.execute and methodWatcher.executeQuery, making it much easier to make it SpotBugs-compliant.

**Review by carolp-503 — APPROVED** [review]
+1

**Pull request merged**